lib/jazz_fingers/prompt.rb

```diff
@@ -28,48 +38,81 @@ def bold_text(text)
       "\001\e[1m\002#{text}\001\e[0m\002"
     end
 
-    def separator
+    def main_separator
       red_text(@separator)
     end
 
-    def name
-      blue_text(@application_name)
+    def wait_separator
+      "*"
+    end
```

Comment:
The naming here is updated for consistency with [Pry's](https://github.com/pry/pry/wiki/Customization-and-configuration#the-prompt) nomenclature:

> Pry can accept two prompt-types for every prompt - the 'main prompt' and the 'wait prompt'. The main prompt is always used for the first line of input; the wait prompt is used in multi-line input to indicate that the current expression is incomplete and more lines of input are required.